integration-tests-anvil/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+services:
+  anvil:
+    image: ghcr.io/foundry-rs/foundry:latest
+    platform: linux/amd64
+    command:
+      - anvil --host 0.0.0.0 --block-time 2
+
+  go-test:
+    image: golang:1.21.4
+    depends_on:
+      - anvil
+    volumes:
+      - ../:/go/src/github.com/status-im/status-go/
+    working_dir: /go/src/github.com/status-im/status-go/api
+    command: |
+        go test -tags gowaku_skip_migrations -run ^TestRouterFeesUpdate github.com/status-im/status-go/api/... -v -- -anvil-host=http://anvil:8545
```

Comment:
right now we are launching a single test here, would be great to replace it with launching all go tests that require anvil if possible